lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -918,8 +916,13 @@ def install_puppetlabs_release_repo_on( hosts, repo = nil, opts = options )
                 variant_url_value = 'cisco-wrlinux'
                 version = '7'
               end
-              remote = "%s/puppetlabs-release%s-%s-%s.noarch.rpm" %
-                [opts[:release_yum_repo_url], repo_name, variant_url_value, version]
+              if repo_name == 'pc1'
+                remote = "%s/puppetlabs-release-%s-%s-%s.noarch.rpm" %
+                  [opts[:release_yum_repo_url], repo_name, variant_url_value, version]
+              else
+                remote = "%s/puppet%s/puppet%s-release-%s-%s.noarch.rpm" %
+                  [opts[:release_yum_repo_url], repo_name, repo_name, variant_url_value, version]
+              end
```

Comment:
on line 907 that I changed, it was previously adding in the `-`. This code assumes users will never want to enable the old `puppetlabs-release-(<platform>-<verision>|<codename>)` packages, so there isn't a case where we need to guard that dash before the repo name. So I figured this would be a little more intuitive. So, that's one caveat to preserving backward compatibility. Users will no longer be able to access those old release packages. Instead, it will default to the newest release packages.

And yes, `release_yum_repo_url/puppet/puppet-release-el-6.noarch.rpm` and `release_yum_repo_url/puppet5/puppet5-release-el-6.noarch.arm` are correct strings. Did you see my email to users explaining the difference between these two repos? You can find it at https://groups.google.com/d/msg/puppet-announce/tKHAKlWnXFA/_Mdsoxo4BAAJ. I didn't send it out internally, though now I'm reconsidering that